### DISCUSSION TITLE
Education benefits

### DISCUSSION THREAD

**Discussion by ToTheMoon365Up** [community/community#137061] [GitHub Education]
### Hi everyone,

I still have not received my education benefits,  5 days has passed

**Comment by davevad93** [comment]
Hi @ToTheMoon365Up ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ToTheMoon365Up was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Meshariabdullahf** [reply]
i need some help to get benefit of student developer pack

**Comment by BlueSky-2024** [comment]
Great discussion on the benefits of education! For additional insights on how educational programs can also lead to practical benefits like reducing electricity bills, check out this resource: [Learn how educational initiatives can contribute to savings on utility costs](https://8171ehsaas-program.pk/reduction-in-electricity-bills/).
